utils/dchemigen/get_image.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
```

Comment:
Needs KOS copyright header.

Reply:
This is missing header where I lifted it from (dcbumpgen).